webscraping/node/courseDataScraper/dataScraper.js

```diff
@@ -0,0 +1,177 @@
+var fs = require('fs');
```

Comment:
why is my version of the datascraper in this PR? Since stu's datascraper in R works and this one is incomplete we shouldn't merge this into master.